### PR TITLE
Set up test dependencies

### PR DESCRIPTION
**Pull request by dhaneshnm**
- Added Enzyme and other test dependencies.
- Added snapshot tests for components.

### CODE REVIEW

File: apps/search-poc/src/components/ResourceDetails/__tests__/EmodelCollection.test.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                id: "id",
+                self: "self_url",
+                name: "name",
+                description: "description"
+            }]
+        }
+    };
+
+
+    // const result = shallow(<EModelDetails {...props} />);
```
Comment (kenjinp) [inline apps/search-poc/src/components/ResourceDetails/__tests__/EmodelCollection.test.tsx]: 🔫

File: apps/search-poc/enzyme.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+declare module 'enzyme-adapter-react-16';
```
Comment (kenjinp) [inline apps/search-poc/enzyme.d.ts]: unnecessary declaration file

File: apps/search-poc/src/components/ResourceDetails/__tests__/EmodelCollection.test.tsx (hunk 1 above)
Comment (dhaneshnm) [reply apps/search-poc/src/components/ResourceDetails/__tests__/EmodelCollection.test.tsx]: Removed.

File: apps/search-poc/enzyme.d.ts (hunk 2 above)
Comment (dhaneshnm) [reply apps/search-poc/enzyme.d.ts]: Removed.

**Pull request merged**